insights/parsers/tests/test_httpd_open_nfs.py

```diff
@@ -0,0 +1,25 @@
+from insights.parsers import httpd_open_nfs
+from insights.parsers.httpd_open_nfs import HttpdOnNFSFilesCount
+from insights.tests import context_wrap
+import doctest
+
+http_nfs = """
+{"http_ids": [1787, 2399], "nfs_mounts": ["/data", "/www"], "open_nfs_files": 1000}
+""".strip()
+
+
+class TestHttpOpenNFS():
```

Comment:
It's not suggested to use a class to apply the unit test in insights-3.0, please remove it.